### DISCUSSION TITLE
Sending logs to opentelemetry

### DISCUSSION THREAD

**Discussion by tushar-gandhi** [open-telemetry/opentelemetry-ruby#1520] [Q&A]
Hello,
We want to forward the Rails application logs to opentelemetry. 
Is it currently supported? If yes then How can I do that?
Thank you

**Comment by arielvalentin** [comment]
The Logs SDK is under development at the moment. 

You may follow along here https://github.com/open-telemetry/opentelemetry-ruby/projects/3

Would you be interested in help us implement the Log SDK?
